src/main/scala/com/danielasfregola/twitter4s/http/clients/rest/search/TwitterSearchClient.scala

```diff
@@ -24,44 +25,45 @@ trait TwitterSearchClient {
-    * @param callback : Optional, by default it is `None`.
-    *                 If supplied, the response will use the JSONP format with a callback of the given name.
-    *                 The usefulness of this parameter is somewhat diminished by the requirement of authentication for requests to this endpoint.
+    * @param geocode          : Optional, by default it is `None`.
+    *                         Returns tweets by users located within a given radius of the given latitude/longitude.
+    *                         The location is preferentially taking from the Geotagging API, but will fall back to their Twitter profile.
+    *                         Note that you cannot use the near operator via the API to geocode arbitrary locations;
+    *                         however you can use this geocode parameter to search near geocodes directly.
+    *                         A maximum of 1,000 distinct “sub-regions” will be considered when using the radius modifier.
+    * @param language         : Optional, by default it is `None`.
+    *                         Restricts tweets to the given language. Language detection is best-effort.
+    * @param locale           : Optional, by default it is `None`.
+    *                         Specify the language of the query you are sending (only `ja` is currently effective).
+    *                         This is intended for language-specific consumers and the default should work in the majority of cases.
+    * @param until            : Optional, by default it is `None`.
+    *                         Returns tweets created before the given date.
+    *                         Keep in mind that the search index has a 7-day limit.
+    *                         In other words, no tweets will be found for a date older than one week.
+    * @param since_id         : Optional, by default it is `None`.
+    *                         Returns results with an ID greater than (that is, more recent than) the specified ID.
+    *                         There are limits to the number of Tweets which can be accessed through the API.
+    *                         If the limit of Tweets has occured since the since_id, the since_id will be forced to the oldest ID available.
+    * @param max_id           : Optional, by default it is `None`.
+    *                         Returns results with an ID less than (that is, older than) or equal to the specified ID.
+    * @param callback         : Optional, by default it is `None`.
+    *                         If supplied, the response will use the JSONP format with a callback of the given name.
+    *                         The usefulness of this parameter is somewhat diminished by the requirement of authentication for requests to this endpoint.
+    * @param tweet_mode       : Selector, when set to Some("extended") prevents tweet text truncating, see https://developer.twitter.com/en/docs/tweets/tweet-updates
```

Comment:
Documentation needs to be updated as it still refers to the implementation using string rather than Enum

Reply:
I'm sorry for the silence, yes, I can update specs and docs. I'll try to add extended tweet support for all respective methods, but I don't know where I should add it exactly. I suppose that all methods returning Tweet should support this params.

Reply:
No worries! Yes, I think for now we can add it to all the methods that return a tweet entity 👍